packages/@dcl/inspector/public/index.html

```diff
@@ -12,6 +12,12 @@
   <noscript>You need to enable JavaScript to run this app.</noscript>
   <div id="root"></div>
   <link href="bundle.css" rel="stylesheet" />
+  <script>
+    const config = '$CONFIG'
+    if (config !== '$CONFIG') {
```

Comment:
Would you like to add a comment that his is meant to be replaced by the Editor? Otherwise just by looking at it seems like the line below the if would never be reached :)

Reply:
added 👍 thanks